Terminal.Gui/Core/View.cs

```diff
@@ -568,7 +568,8 @@ void SetTabIndex ()
 				}
 
 				width = value;
-				if (autoSize && value.Anchor (0) != TextFormatter.Size.Width) {
+				if (autoSize && value.Anchor (0) != TextFormatter.Size.Width
+					- (TextFormatter.Text.Contains (HotKeySpecifier) ? 1 : 0)) {
```

Comment:
Not sure what HotKeySpecifieer has to do with  this. 

In the Wizard.cs bug, removing the Hotkeyspecifier from "Fi_nished" (to "Finished") has no impact on autosize being turned off.

Reply:
Yes it has to do. See the changes please.